File: Programming Basics with JS/T05 - For Loop/exercise/06. Oscars.js
function oscars(input) {
    let name = input[0];
    let points = Number(input[1]);
    let judges = Number(input[2]);
    let judgeName, judgePoints;
    let index1 = 3, index2 = 4;

    for (let i = 0; i < judges; i++) {
        judgeName = input[index1];
        judgePoints = Number(input[index2]);
        points = points + ((judgeName.length * judgePoints) / 2);

        if (points > 1250.5) {
            console.log(`Congratulations, ${name} got a nominee for leading role with ${points.toFixed(1)}!`);
            return;
        }
        index1 += 2;
        index2 += 2;
    }
    console.log(`Sorry, ${name} you need ${(1250.5 - points).toFixed(1)} more!`);
}
oscars(['Zahari Baharov', '205', '4',
    'Johnny Depp', '45',
    'Will Smith', '29',
    'Jet Lee', '10',
    'Matthew Mcconaughey', '39']);

oscars(["Sandra Bullock", "340", "5",
    "Robert De Niro", "50",
    "Julia Roberts", "40.5",
    "Daniel Day-Lewis", "39.4",
    "Nicolas Cage", "29.9",
    "Stoyanka Mutafova", "33"]);

File: Programming Basics with Python/T05 - For Loop/exercise/06. Oscars.py
actor_name = input()
academy_points = float(input())
judges_count = int(input())
total_judge_points = 0
total_points = 0

for number in range(judges_count):
    judge_name = input()
    judge_points = float(input())
    total_judge_points += len(judge_name) * judge_points / 2
    total_points = academy_points + total_judge_points

    if total_points >= 1250.5:
        print(f"Congratulations, {actor_name} got a nominee for leading role with {total_points:.1f}!")
        break

if total_points < 1250.5:
    needed_points = 1250.5 - total_points
    print(f"Sorry, {actor_name} you need {needed_points:.1f} more!")